modules/ws-client/src/main/scala/trading/client/View.scala

```diff
@@ -0,0 +1,140 @@
+package trading.client
+
+import cats.syntax.show.*
+
+import trading.domain.*
+import trading.client.ui.*
+
+import tyrian.*
+import tyrian.Html.*
+
+def render(model: Model): Html[Msg] =
+  val tableHidden: Attr[Nothing] =
+    if model.alerts.isEmpty then hidden else attribute("foo", "")
+
+  div(`class` := "container")(
+    genericErrorAlert(model),
+    subscriptionSuccess(model),
+    unsubscriptionSuccess(model),
+    h2(attribute("align", "center"))(text("Trading WS")),
+    div(`class` := "input-group mb-3")(
+      input(
+        `type` := "text",
+        id     := "symbol-input",
+        autoFocus,
+        placeholder := "Symbol (e.g. EURUSD)",
+        onInput(s => Msg.SymbolChanged(s)),
+        onKeyDown(subscribeOnEnter),
+        attribute("value", if model.symbol.show == "XXXXXX" then "" else model.symbol.show)
+      ),
+      div(`class` := "input-group-append")(
+        button(
+          `class` := "btn btn-outline-primary btn-rounded",
+          onClick(Msg.Subscribe)
+        )(
+          text("Subscribe")
+        )
+      )
+    ),
+    div(id := "sid-card", `class` := "card")(
+      div(`class` := "sid-body")(
+        renderTradeStatus(model.tradingStatus),
+        span(text(" ")),
+        renderSocketId(model.socketId)
+      )
+    ),
+    p(),
+    table(`class` := "table table-inverse", tableHidden)(
+      thead(
+        tr(
+          th(text("Symbol")),
+          th(text("Bid")),
+          th(text("Ask")),
+          th(text("High")),
+          th(text("Low")),
+          th(text("Status")),
```

Comment:
You should find you can drop the `text` to `th("Status")` if you want to.

Reply:
Sweet!